modified_files/net/net.gypi

```diff
@@ -817,6 +817,8 @@
       'http2/hpack/decoder/hpack_block_decoder.h',
       'http2/hpack/decoder/hpack_decoder_string_buffer.cc',
       'http2/hpack/decoder/hpack_decoder_string_buffer.h',
+#      'http2/hpack/decoder/hpack_decoder_tables.cc',
```

Comment:
So why are we adding the files in this pull request if they aren't being used? Either we should add the files and uncomment the net.gypi lines such that they get built/tested, or we don't add the files.